### PR TITLE
Disable BigQuery test by default

### PR DESCRIPTION
**Pull request by RVRhub**

### CODE REVIEW

File: datafaker-gen-examples/datafaker-gen-bigquery/pom.xml [hunk 1]
```diff
@@ -161,4 +161,48 @@
...
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>3.3.0</version>
```
Comment (snuyanzin) [inline datafaker-gen-examples/datafaker-gen-bigquery/pom.xml]: Could you please move version to properties
like it is done for datafaker https://github.com/datafaker-net/datafaker/blob/f8b2473528c3dbe0a90c835202c3e1bef0d5be33/pom.xml#L46

File: datafaker-gen-examples/datafaker-gen-bigquery/pom.xml [hunk 2]
```diff
@@ -161,4 +161,48 @@
...
+                        <configuration>
+                            <excludes>
+                                <exclude>**/net/datafaker/datafaker_gen/sink/**/*Test.java</exclude>
```
Comment (snuyanzin) [inline datafaker-gen-examples/datafaker-gen-bigquery/pom.xml]: Do we really need to exclude all or just `BigQueryConnectionConfigurationTest` is enough?

Comment (RVRhub) [reply datafaker-gen-examples/datafaker-gen-bigquery/pom.xml]: No. I exclude just one BigQuerySinkIntegrationTest.

File: datafaker-gen-examples/datafaker-gen-bigquery/pom.xml [hunk 3]
```diff
@@ -147,14 +147,26 @@
                 <configuration>
                     <useManifestOnlyJar>false</useManifestOnlyJar>
                     <argLine>${surefire.argline}</argLine>
-                    <skipTests>${skipBigQueryTests}</skipTests>
+                    <skipTests>${skipBigQueryIntegrationTests}</skipTests>
```
Comment (snuyanzin) [inline datafaker-gen-examples/datafaker-gen-bigquery/pom.xml]: hm... this should be for `integration-test`  phase IIUC
or did I miss anything?

**Pull request merged**